src/index.scss

```diff
@@ -0,0 +1,16 @@
+/* Remove overrides once Carbon bugs are fixed upstream. */
+
+/*
+The React <Content /> component uses the `main` element which IE11 doesn't recognize
+
+@link https://github.com/carbon-design-system/carbon/issues/2789
+*/
+
+@import 'carbon-components/scss/globals/scss/styles.scss';
+main {
+  display: block;
+}
+
+$feature-flags: (
```

Comment:
Per instructions https://www.carbondesignsystem.com/tutorial/react-step-1/#add-ui-shell this should come before the styles import